.github/workflows/CI.yml

```diff
@@ -68,9 +68,9 @@ jobs:
             using Pkg
             Pkg.develop(PackageSpec(path=pwd()))
             Pkg.instantiate()'
-            #- name: "Run doctests"
-            #run: |
-            #julia --project=docs --color=yes test/doctest.jl
+            - name: "Run doctests"
```

Comment:
This line is indented wrong (too deep) hence the error. Make sure to indent it identically to hhe preceding step

Reply:
Thank you - that indeed seemed to be the problem.